ee_tests/docker-entrypoint.sh

```diff
@@ -1,10 +1,5 @@
 #!/bin/bash
 
-
-echo -n Updating Webdriver and Selenium...
-webdriver-manager update
-webdriver-manager update --versions.chrome 2.33
-
 ## FIXME: Firefox complains about a missing machine-id file. So I set a random one
 echo 8636d9aff3933f48b95ad94891cd1839 > /var/lib/dbus/machine-id
```

Comment:
1. Is it still needed?
2. If yes, can we put it directly into `Dockerfile.builder`?